tools/cli/app_test.go

```diff
@@ -200,6 +200,69 @@ var describeDomainResponseServer = &types.DescribeDomainResponse{
 	},
 }
 
+// This is used to mock the domain deletion confirmation
+func (s *cliAppSuite) mockStdinInput(input string) func() {
+	oldStdin := os.Stdin
+	r, w, _ := os.Pipe()
+	os.Stdin = r
+
+	go func() {
+		w.Write([]byte(input + "\n"))
```

Comment:
```suggestion
		wrErr := w.Write([]byte(input + "\n"))
		s.NoError(wrErr)
```
Also handle an error here.